app/src/main/java/com/example/seonoh/seonohapp/network/RetrofitCreator.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.seonoh.seonohapp.network
+
+import com.example.seonoh.seonohapp.BuildConfig
+import com.facebook.stetho.okhttp3.StethoInterceptor
+import com.ihsanbal.logging.Level
+import com.ihsanbal.logging.LoggingInterceptor
+import okhttp3.OkHttpClient
+import okhttp3.internal.platform.Platform
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+const val BASE_URL = "https://api.upbit.com/"
+class RetrofitCreator {
+
+    companion object{
+        fun createNet() : Api{
+            val client = OkHttpClient.Builder()
```

Comment:
```suggestion
            val okClient = OkHttpClient.Builder()
                .addInterceptor(
                    LoggingInterceptor.Builder()
                        .loggable(BuildConfig.DEBUG)
                        .setLevel(Level.BASIC)
                        .log(Platform.INFO)
                        .request("OH_req")
                        .response("OH_res")
                        .build()
                )
                .addNetworkInterceptor(StethoInterceptor())
                .build()
```
변수를 하나 더 선언할 필요없이 한번에 설정